src/test/kotlin/interactor/GetAverageFruitAndVegetablesInteractorTest.kt

```diff
@@ -0,0 +1,38 @@
+package interactor
+
+import dataSource.utilis.FakeData
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class GetAverageFruitAndVegetablesInteractorTest {
```

Comment:
you need more test case "one not enough" 
one case 
1. return empty list if all cities have no average salary.
try add more